bookshelf/web/index.php

```diff
@@ -24,8 +24,14 @@
 
 $app = require __DIR__ . '/../src/app.php';
 require __DIR__ . '/../src/controllers.php';
-$app['bookshelf.model'] = new CloudSql();
+
```

Comment:
I feel like this diff shows a diff against older file contents. I don't know why.